### ISSUE TITLE
Disable Finger Tracking Toggle Not Working

### ISSUE THREAD

**Issue by HATExRecon** [bdunderscore/attach-to-me#38]
Sorry to make another report, but the check box to disable finger tracking and attachmetn doesn't seem to work. Looking at the code, it looks like the bool is named differently in the attachableconfig script, but I'm not sure if that's the issue, as changing it didn't seem to make a difference

**Comment by bdunderscore** [comment]
Yeah, I noticed that when working on U#1.0 support, I'll take a look this weekend.
